test/e2e/kuma_deploy_global_remote_test.go

```diff
@@ -131,52 +132,12 @@ metadata:
 			}
 		}
 		Expect(found).To(BeTrue())
-	})
 
-	It("should deploy Remote and Global on 2 clusters and sync dataplanes", func() {
-		// given
-		namespace := func(namespace string) string {
-			return fmt.Sprintf(`
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: %s
-`, namespace)
-		}
-		dp := func(cluster, namespace, name string) string {
-			return fmt.Sprintf(`
-apiVersion: kuma.io/v1alpha1
-kind: Dataplane
-mesh: default
-metadata:
-  name: %s
-  namespace: %s
-spec:
-  networking:
-    address: 192.168.0.1
-    inbound:
-      - port: 12343
-        tags:
-          kuma.io/service: backend
-          kuma.io/zone: %s
-    outbound:
-      - port: 1212
-        tags:
-          kuma.io/service: web
-`, name, namespace, cluster)
-		}
```

Comment:
this DP was not valid DP due to service tag value. Instead, we can just see if the DP of the app is synced